parser/result.go

```diff
@@ -486,9 +486,10 @@ func (r *result) asMethodDescriptor(node *ast.RPCNode) *descriptorpb.MethodDescr
 	return md
 }
 
-func (r *result) asEnumDescriptor(en *ast.EnumNode, handler *reporter.Handler) *descriptorpb.EnumDescriptorProto {
+func (r *result) asEnumDescriptor(en *ast.EnumNode, syntax syntaxType, handler *reporter.Handler) *descriptorpb.EnumDescriptorProto {
 	ed := &descriptorpb.EnumDescriptorProto{Name: proto.String(en.Name.Val)}
 	r.putEnumNode(ed, en)
+	rsvdNames := map[string]int{}
```

Comment:
Small nit - looks like we are only using the count to determine if it is >1. We could tweak to use struct{} value instead.

Reply:
Okay, I changed this so it actually stores the position of the first declaration and can report that in the error message.